x-pack/plugins/uptime/public/components/overview/alerts/toggle_alert_flyout_button.test.tsx

```diff
@@ -15,6 +17,18 @@ import {
 import { ToggleAlertFlyoutButtonComponent } from './toggle_alert_flyout_button';
 import { ToggleFlyoutTranslations } from './translations';
 
+const mockStartServices = mockCoreMock.createStart();
+jest.mock('../../../hooks/use_kibana', () => ({
+  useKibanaContextForPlugin: () => ({
+    services: {
+      ...mockStartServices,
+      observability: {
+        useRulesLink: jest.fn().mockImplementation(() => ({ href: 'newRuleLink' })),
```

Comment:
we have a common place for this in `../../../lib/helper/rtl_helpers`

can you please make this part of that helper?

that will make sure, whenever in uptime tests plugin we use useKibana, we have access to that helper